### PR TITLE
My_solutions_for_exercise_10

### PR DESCRIPTION
**Pull request by kingsedem**
Kindly see the solutions for exercise 10. 

Note: All comments were cloned by default from the exercise.

### CODE REVIEW

**Comment by globalworming** [comment]
> Note: All comments were cloned by default from the exercise.

:grin: no worries, i don't care about those. i basically just look at code that changed

File: src/test/java/com/serenitydojo/WhenWritingReadableAssertions.java [hunk 1]
```diff
@@ -38,55 +39,55 @@ public void whenANumberIsInACertainInterval() {
...
+        assertThat(roughlyOneHundred).isCloseTo(100, Percentage.withPercentage(10));
     }
 
     @Test
     public void whenAStringIsEqualToAnother() {
         String color = "red";
 
         // Check that the string has a value of 'red'
-        assertThat(true).isFalse();
+        assertThat(color).contains("red");
```
Comment (globalworming) [inline src/test/java/com/serenitydojo/WhenWritingReadableAssertions.java]: contains works.. but is not intended i think
`assertThat("darkred").contains("red");` is also true and we are asking about equality here. so `isEqual` is probably what's asked for here, right?

File: src/test/java/com/serenitydojo/WhenWritingReadableAssertions.java [hunk 2]
```diff
@@ -95,38 +96,40 @@ public void whenAListContainsAValue() {
...
+
     }
 
     @Test
     public void whenAListContainsExactlyASetOfValuesInAnyOrder() {
         List<String> colors = Arrays.asList("red","green","blue");
 
         // Check that the list contains "red","blue" and "green" in any order
-        assertThat(true).isFalse();
+        assertThat(colors).containsExactlyInAnyOrder("red", "green", "blue");
```
Comment (globalworming) [inline src/test/java/com/serenitydojo/WhenWritingReadableAssertions.java]:
```suggestion
        assertThat(colors).containsExactlyInAnyOrder("red", "blue", "green");
```
the point is to showcase that the order can be mangled but it still asserts successfully

File: src/test/java/com/serenitydojo/WhenWritingReadableAssertions.java (hunk 1 above)
Comment (kingsedem) [reply src/test/java/com/serenitydojo/WhenWritingReadableAssertions.java]: Yeah, just saw it now.  you are correct.  I'll update it.

File: src/test/java/com/serenitydojo/WhenWritingReadableAssertions.java (hunk 2 above)
Comment (kingsedem) [reply src/test/java/com/serenitydojo/WhenWritingReadableAssertions.java]: Wow, so true. I missed this. I'll correct it.

**Pull request merged**